docs/solidity-to-cadence.mdx

```diff
@@ -0,0 +1,447 @@
+scopes defined for storage paths and Capabilities themselves align precisely with
+[PublicAccount](language#accounts#publicaccount)/[AuthAccount](language#accounts#authaccount) account scopes.
+
+Any account can get access to an account's public Capabilities. Public capabilities are created using public paths,
+i.e. they have the domain `public`. For example, all accounts have a default public capability linked to the
+`FLOWToken.Vault` Resource. This vault is exposed as a public capability, scoped to the `receiver` resource interface,
+to allow any account to `borrow()` the Vault to make a `deposit()`. Public capabilities can be obtained from both
+authorized accounts (`AuthAccount`) and public accounts (`PublicAccount`).
+
+Private capabilities are specifically granted to accounts. They are created using private paths, i.e. they have the
+domain `private`. After creation, they can be obtained from authorised accounts (`AuthAccount`) but not public
+accounts (`PublicAccount`). To share a private Capability with another account, the owning account must `publish` it
+to another account which places in the [account inbox](language#accounts#account-inbox). The recipient can later
+claim the Capability from the account inbox using then `claim` function.
+
+Capabilities can be `unpublished` and can also be [revoked](design-patterns#capability-revocation) by the creating
+account.
+
+To aid automation, events are emitted for `publish`, `claim` and `unpublish` actions completed for a Capability.
+
+Detailed information can be found in [Capabilities](language#capability-based-access-control).
+
+## Hygiene factors for protecting value
+
+While capabilities grant account access to a protected resource, it's still necessary to impose controls on value
+accessed through them. For example, if your use-case requires delegating access to a `FLOWToken.Vault` to
+`withdraw()` funds, it's important to limit the amount. Tokens implementing FT/NFT standards are the primary type of
+value being exchanged by accounts on Flow. The standard provides the primitives needed to implement capability
+limiting best-practices.
+
+### Token isolation
+
+All FTs reside in a `Vault` Resource and each different FT will exist as a separate `Vault` in an account. Similarly,
+all NFTs implement a `Collection` Resource, in which those NFTs held by an account for that collection are stored.
+
+Whenever access to the `withdraw()` function has to be delegated to another account, the simplest way to limit how
+many tokens of a given type can be withdrawn is to create a new `Vault` Resource for that token type and move a smaller
+amount of the tokens in the main token `Vault`. A capability is then linked to that `Vault` instance before being
+made available to another account.
+
+A similar pattern can be used for NFTs, where a new `Collection` Resource can be created into which only those NFTs
```

Comment:
This could be done, but I haven't really seen any piece of code doing it. I believe it's really well explained just with the FT paragraph